### ISSUE TITLE
Though defaultMultipleSelection is true still multiple selection is not enabled

### ISSUE THREAD

**Issue by saurabhbajajcode** [Yummypets/YPImagePicker#524]
**Describe the bug**
When we set `YPImagePickerConfiguration().library.defaultMultipleSelection = true` still multiple selection is not enabled by default. 

**To Reproduce**
1. Create YPImagePicker object. You can use following code:
```
@IBAction func uploadImageTapped(_ sender: Any) {
        var config = YPImagePickerConfiguration()
        config.library.mediaType = .photo
        config.showsPhotoFilters = false
        config.shouldSaveNewPicturesToAlbum = false
        config.startOnScreen = .library
        config.screens = [.library, .photo]
        config.showsCrop = .none
        config.wordings.libraryTitle = "Gallery"
        config.hidesStatusBar = false
        config.hidesBottomBar = false
        config.maxCameraZoomFactor = 2.0
        config.library.maxNumberOfItems = allowedMaxTotalImages
        config.gallery.hidesRemoveButton = false
        config.library.preselectedItems = selectedYPMediaItems
        config.library.skipSelectionsGallery = true
        config.library.defaultMultipleSelection = true

        let picker = YPImagePicker(configuration: config)
        picker.didFinishPicking { [unowned picker] items, cancelled in
            if cancelled {
                picker.dismiss(animated: true, completion: nil)
                return
            }
            self.selectedYPMediaItems = items
            self.removeAllButton.isHidden = self.selectedYPMediaItems.isEmpty
            self.setNextButtonTitleForImageViewScreen()
            picker.dismiss(animated: true, completion: nil)
        }
        present(picker, animated: true, completion: nil)
 }
```
2. Call above function on tap of a button
3. Select only 1 image. Tap Next.
4. Again open image picker.
5. When image picker is displayed you will observe that multiple selection is not enabled even though defaultMultipleSelection was set as true. Also, the previous selected image is not highlighted.
Refer attached gif.

**Expected behavior**
When defaultMultipleSelection is set as true then multiple selection should get enabled irrespective of number of items previously selected.

**Screenshots**
This screenshot and gif is after setting defaultMultipleSelection as true.

![Screenshot 2020-07-13 at 12 17 23 AM](https://user-images.githubusercontent.com/10595656/87254185-53524980-c49e-11ea-9554-ff9984d266c1.png)

![YPImagePicker_Default_Multiple_Selection_Issue](https://user-images.githubusercontent.com/10595656/87254680-d628d380-c4a1-11ea-80eb-6130e49ab0db.gif)

**Environment (please complete the following information):**
 - Device: Any device
 - OS: Any OS
 - Xcode Version Any version
 - Swift Version Any version

**Installation Type**
 - Cocoapods: I have tried only with CocoaPods. 

**Additional context**
I am using release 4.2.0

**Comment by saurabhbajajcode** [comment]
I do not have access to link PR #525 with this issue.

**Comment by s4cha** [comment]
Hi @saurabhbajajcode,
Thank you so much for noticing that, documenting it thoroughly with video gif (amazing!) and issuing a clean PR that fixes it. 
This is really splendid !

**Comment by heitara** [comment]
@s4cha shall we close this, once it's already merged?

**Comment by s4cha** [comment]
Yep, this should be fixed as of 4.3.0 and coming 4.3.1 :)
